client-side/src/stories/DateTime/DateInputs.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import PropTypes from 'prop-types';
+
+import { DemoContainer } from '@mui/x-date-pickers/internals/demo';
+import { LocalizationProvider } from '@mui/x-date-pickers-pro/LocalizationProvider';
+import { AdapterDayjs } from '@mui/x-date-pickers-pro/AdapterDayjs';
```

Comment:
please import all the mui date pickers in one line, something like:
import {  LocalizationProvider, AdapterDayjs } from '@mui/x-date-pickers